tests/xfstests/partition.pm

```diff
@@ -261,6 +261,13 @@ sub format_with_options {
         format_partition($part, 'xfs', options => '-f -m crc=1,reflink=0,rmapbt=0, -i sparse=0 -lsize=100m');
         script_run("echo 'export XFS_MKFS_OPTIONS=\"-m crc=1,reflink=0,rmapbt=0, -i sparse=0 -lsize=100m\"' >> $CONFIG_FILE");
     }
+    # In case to test different mkfs.btrfs options
+    # $XFSTEST_MKFS_OPTION: options for mkfs.btrfs
```

Comment:
nit: if used just for btrfs, shouldn't the variable be `XFSTEST_MKFS_BTRFS_OPTION`?

Reply:
yes i thought about it. I think both xfs and btrfs can use this variable, and $filesystem can used to determine fs type.

Reply:
I'm sorry I overlooked xfs uses it as well. Than `XFSTEST_MKFS_OPTION` is correct variable name.